### PR TITLE
Doc/13 templates for design docs

### LINKED ISSUE #13: Use specific templates for planning and design
The project should have elements such as user stories, use cases, requirements, user journeys, and personas.

Each of these elements should be consistent in their own structure, so in order to accomplish it, elaborate adequate templates to be used when describing these elements.

Specifically, user stories, user journeys and personas might beneficiate from a template because some sort of table-like structure would be appropriate, but use cases and requirements will probably just be a list, so it won't require a template since it will be easy to keep consistency.

This is necessary for #7

### PR DESCRIPTION
**Pull request by Anglepi**
- Added a template to fill for each user story.
- Added a template to fill for each persona.
- _**New**_ Added user journey template

closes #13.

# Review tasks

- [x] Add user journey template, since #13 has been updated

### CODE REVIEW

**Comment by Anglepi** [comment]
Spellchecker will fail until #11 is merged since it contains the necessary to add exceptions.

I can either merge this as it is and add the exceptions later or wait until #11 is merged, then merge this with main and update the exceptions

**Comment by Anglepi** [comment]
@JJ Esto está listo ya para revisar, son tres plantillas

**Review by JJ — APPROVED** [review]
Not a lot to review here. Looks good to me.

**Pull request merged**
